gubernator/templates/build.html

```diff
@@ -7,7 +7,7 @@
 % endif
 % endblock
 % block header
-<h1>{% if pr %}<a href="/pr/{{pr}}">PR #{{pr}}</a> {% endif %}
+<h1>{% if pr %}<a href="/pr/{{pr_path}}{{pr}}">{% if repo != "kubernetes/kubernetes" %}{{repo}} {% endif %}PR #{{pr}}</a> {% endif %}
```

Comment:
I would actually just always say the repo, rather than special-casing `kubernetes/kubernetes` here. Same for PRs.